### DISCUSSION TITLE
v2.7.6

### DISCUSSION THREAD

**Discussion by Limraj** [SCADA-LTS/Scada-LTS#2671] [General]
The changelist is listed below:

### 📄️ GitHub issues/pullrequest list overview:
1. #607 Fixed validation Id field in Event Search 
2. #2190 Fixed feature 'Apply new properties basing on' for property 'Event text renderer properties'
3. #2364 Fixed the details in the Data Sources list
4. #2580 Fixed footer in reportChart.ftl
5. #2593 Fixed flags on Alarm list
6. #2596 The selection of images in the binary graphic component is not shown to the user (#2520)
7. #2599 Component "chart comparator" does not show charts
8. #2642 Fixed creating Alphanumeric data point in Virtual Data Source in new UI
9. #2666 Fixed visible 'Access User' and 'Views' in 'Point Details'

<hr /><em>This discussion was created from the release <a href='https://github.com/SCADA-LTS/Scada-LTS/releases/tag/v2.7.6'>v2.7.6</a>.</em>

**Comment by fabiodurao** [comment]
Could you update the installers for Windows and Linux with this version? Grateful.

**Comment by Limraj** [comment]
Hi @fabiodurao, 
should already be updated. We wanted to do this with version 2.7.6.1.

Regards,
Kamil Jarmusik
